iree/testing/vulkan/CMakeLists.txt

```diff
@@ -35,13 +48,11 @@ iree_cc_library(
   SRCS
     "vulkan_gui_util.cc"
   DEPS
-    dear_imgui::dear_imgui
-    dear_imgui::impl_sdl
-    dear_imgui::impl_vulkan
+    imgui::imgui
```

Comment:
could these just be `imgui` and `SDL2`?

Reply:
short answer: no

longer answer: maybe, those targets aren't directly exposed by the installed libraries, but we have some special handling in `iree_cc_library` that is similar: https://github.com/google/iree/blob/a49b97b8759436d0a5ece44689a7ca76ddbf3f30/build_tools/cmake/iree_cc_library.cmake#L80-L82

`...\vcpkg\installed\x64-windows\share\imgui\imgui-config.cmake`:
```
# Create imported target imgui::imgui
add_library(imgui::imgui STATIC IMPORTED)
```

`...\vcpkg\installed\x64-windows\share\sdl2\SDL2Targets.cmake`:
```
# Create imported target SDL2::SDL2
add_library(SDL2::SDL2 SHARED IMPORTED)
```

Reply:
oh cool - I didn't know that they doubled them up like that - that's awesome then and makes us seem less weird :P